src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -433,5 +433,14 @@ private static void NewMethod(out Construct obj1, out Construct obj2, out Constr
 
 compareTokens: false);
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsExtractMethod)]
+        public async Task TestTuple()
+        {
+            await TestAsync(
+@"class Program { static void Main ( string [ ] args ) { [| (int, int) x = (1, 2); |]  System . Console . WriteLine ( x.Item1 ); } } ",
```

Comment:
@CyrusNajmabadi Is there a reason for the weird spacing in those tests? I just mimicked other tests in the file...